src/overlays/actors/ovl_En_Rz/z_en_rz.c

```diff
@@ -42,80 +39,646 @@ const ActorInit En_Rz_InitVars = {
     (ActorFunc)EnRz_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80BFCCF4 = {
-    { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_ENEMY, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_1, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
+TexturePtr D_80BFCCE0[] = { object_rz_Tex_00BC50, object_rz_Tex_00C190, object_rz_Tex_00C590 };
+
+TexturePtr D_80BFCCEC = object_rz_Tex_00C990;
+
+TexturePtr D_80BFCCF0 = object_rz_Tex_00CD90;
```

Comment:
From usage these are very likely single array (or it would likely use the textures directly).